models/order.go

```diff
@@ -245,6 +257,16 @@ func init() {
 
 	rand.Seed(time.Now().UnixNano())
 
+	// If there is a mongo-password secret in the flexvol mount reset mongoPassword var 
+	if mongoPassword == "" {
```

Comment:
This will be executed thousands of times under load test, is it possible to move this to the initialization?